src/views/Tenant/TenantView.js

```diff
@@ -165,9 +165,9 @@ const Tenant = () => {
       comp: <div />,
     },
     {
-      key: "unit",
-      label: "Unit",
-      value: property.unit,
+      key: "unitNum",
+      label: "Total Units",
```

Comment:
I think "Unit" was probably the correct label for this one.
```suggestion
      label: "Unit",
```
This confusion is exactly why we didn't set this up right in the first place 😂